internal/manifest/compose/parse.go

```diff
@@ -6,6 +6,7 @@ import (
 	"github.com/goccy/go-yaml"
 )
 
+// TODO: Consider whether this representation is useful or if we can just use `compose.Compose`,
```

Comment:
it's trivial to recreate - so if you're not using it now, just go ahead and delete it